### DISCUSSION TITLE
Kohya HRFix. How do i use it? Or HRFix in general???

### DISCUSSION THREAD

**Discussion by heyitshestia** [lllyasviel/stable-diffusion-webui-forge#502] [Q&A]
Hey! I'm trying to use Kohya for higher resolution images, but the integrated one is so different from the one you can install through extensions making it hard to find any guides that are applicable. Could someone explain to me how to use the integrated kohya?

Also, could someone tell me some good settings for HiresFix? I know my way around it in A1111, but in forge it seems to straight up demolish any generated image no matter the upscaler and/or settings i have been trying to use, and i don't know if its just not working or if i'm doing something wrong...

**Comment by ghost** [comment]
**Well, I hope that this is helpful.**

I'm still an amateur, _and_ I have very low VRAM, but have been using Forge for a while now.

When it comes to Hirex.Fix, I've gotten good results using:

Sampling steps: 20
CFG Scale: 7
Denoising Strength: 0.6
Hires Steps: 12

I used the DPM++ 2M SDE (Karras) sampler, but I don't think that makes a difference.

The most _important_ part that I have found?  Use the "Latent" upscaler.  I want to use the R-ESRGAN 4+ because it's supposed to be superior for photorealistic images.  However, no matter _which_ settings I use, if I use R-ESRGAN 4+, all that I'm left with is a totally black image staring back at me.  I haven't experimented with any other upscalers yet.  

BUT --  _Latent_ seems to be the key right now.  Hopefully that will change.

Take care and good luck to you!

**Comment by thiagojramos** [comment]
Does it work with Flux?

**Comment by ghost** [comment]
Good question, I have no idea... I don't have enough VRAM to play with anything bigger than SD 1.5, unfortunately.
________________________________
